fontforgeexe/uiutil.c

```diff
@@ -593,13 +604,7 @@ return( true );
 	    }
 	}
       break;
-      case et_selclear:
-	errdata.start_l = errdata.end_l = -1;
```

Comment:
I believe this was klipper taking over the selection and sending FF this message, which clears out the selected text. But I think it's better to just leave the selection as-is until the user manually clears it by clicking again in the warnings dialogue.

Reply:
Oh! Yes, I am guilty of using KDE. Sorry about that! Didn't think it mattered.